src/jit/gentree.cpp

```diff
@@ -6246,15 +6246,310 @@ bool GenTree::TryGetUse(GenTree* def, GenTree*** use)
     assert(def != nullptr);
     assert(use != nullptr);
 
-    for (GenTree** useEdge : UseEdges())
+    switch (OperGet())
+    {
+        // Leaf nodes
```

Comment:
Just to clarify, by "leave this as-is" I mean "leave this PR as-is and hopefully get it merged" :)